bugbear.py

```diff
@@ -677,7 +683,7 @@ def check_for_b017(self, node):
                     and item_context.func.id == "raises"
                     and isinstance(item_context.func.ctx, ast.Load)
                     and "pytest.raises" in self._b005_imports
-                    and "match" not in [kwd.arg for kwd in item_context.keywords]
+                    and "match" not in (kwd.arg for kwd in item_context.keywords)
```

Comment:
Just making tuples since they don't need to be immutable?

Reply:
Actually not a tuple, just parenthesizing the generator:

```
>>> x = [i for i in range(3)] 
>>> x
[0, 1, 2]
>>> x = (i for i in range(3))
>>> x
<generator object <genexpr> at 0x0000021C2DFCAA80>
```

Just a tiny efficiency improvement, rather than having iterate the generator to construct a list, then iterate through the list to find the thing... just iterate the generator directly